example/ipfragreass/odp_ipfragreass.c

```diff
@@ -230,8 +230,9 @@ int main(void)
 	odp_pool_t fragment_pool;
 	odp_shm_t shm;
 	odp_cpumask_t cpumask;
-	odph_odpthread_t threads[MAX_WORKERS];
-	odph_odpthread_params_t thread_params;
```

Comment:
Should we deprecate these types and related functions in the helper API?

Reply:
I added the `@deprecated` tag to the documentation of these types. The functions are already tagged. We'll think about actual deprecation later.